core/src/test/java/org/elasticsearch/script/ScriptTests.java

```diff
@@ -52,6 +54,21 @@ public void testScriptParsing() throws IOException {
         }
     }
 
+    public void testScriptSimpleParsing() throws IOException {
+        String simpleScript = "{\"script\":\"_score + doc['field']\"}";
```

Comment:
@clintongormley @rjernst Is it isn't likely we get #20122 as part of ES 5? If we think it is likely then I'll close this PR. Otherwise we should push this PR as it fixes a bug.